omod/src/main/java/org/openmrs/module/spa/extension/html/AdminList.java

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
Please remove all classes that are not explicitly needed.  The SDK creates a bunch of "helpful" base classes for a common use case, but these should be deleted before committing if they are not needed.